src/py/flwr/common/constant.py

```diff
@@ -40,15 +40,14 @@
 # Addresses
 # SuperNode
 CLIENTAPPIO_API_DEFAULT_ADDRESS = "0.0.0.0:9094"
-# SuperExec
-EXEC_API_DEFAULT_ADDRESS = "0.0.0.0:9093"
 # SuperLink
 DRIVER_API_DEFAULT_ADDRESS = "0.0.0.0:9091"
 FLEET_API_GRPC_RERE_DEFAULT_ADDRESS = "0.0.0.0:9092"
 FLEET_API_GRPC_BIDI_DEFAULT_ADDRESS = (
     "[::]:8080"  # IPv6 to keep start_server compatible
 )
 FLEET_API_REST_DEFAULT_ADDRESS = "0.0.0.0:9093"
+EXEC_API_DEFAULT_ADDRESS = "0.0.0.0:9095"
```

Comment:
@Robert-Steiner do we need to change anything else to make it work with the new default port number?

Reply:
I changed the default port here b/c it conflicts with the default port for Rest-based Fleet API. We can also updated the default port for that one and keep the default port for exec API unchanged if it's easier to do so.

Reply:
I'm fine with `9095`. We need to update the Docker docs anyway as we need to remove SuperExec from it.

Reply:
@panh99 great idea, let's change the default port for the REST API (which is not commonly used at this point)